app/globals.css

```
@@ -448,3 +450,111 @@
   --color-tier-6-bg: theme(colors.blue.800);
   --color-tier-6-text: theme(colors.blue.200);
 }
+
+@keyframes fadeIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)